WordPress/src/main/java/org/wordpress/android/ui/voicetocontent/VoiceToContentDialogFragment.kt

```diff
@@ -78,17 +83,16 @@ class VoiceToContentDialogFragment : BottomSheetDialogFragment() {
                 }
 
                 override fun onSlide(bottomSheet: View, slideOffset: Float) {
-                    // Handle the slide offset if needed
+                    // no op
                 }
             })
 
-            // Disable touch interception by the bottom sheet to allow nested scrolling
+            // Disable touch interception by the bottom sheet to allow nested scrolling for landscape and small screens
             bottomSheet.setOnTouchListener { _, _ -> false }
         }
 
         // Observe the ViewModel to update the cancelable state of closing on outside touch
         viewModel.isCancelableOutsideTouch.observe(this) { cancelable ->
-            Log.i(javaClass.simpleName, "***=> disable outside touch")
```

Comment:
Good to remove this log. I also thought it was unnecessary. 👍🏻

Reply:
:) My final PR cleans up other log lines and debug comments/helpers, too!

Reply:
Thank you!